woocommerce/woocommerce-android
# info fragment user elig error menu usage

Closes: #7266 
Closes: #7268


### Description

The PR updates fragment-ktx library and replaced deprecated API usage related to the menus in 2 fragments:
* InfoScreenFragment
* UserEligibilityErrorFragment

### Testing instructions

* Click on learn more in the order details to check that there is no user-facing changes on `InfoScreenFragment`
<img width="462" alt="image" src="https://user-images.githubusercontent.com/4923871/187415901-c9ec7f0a-fdc3-4f8a-a975-11f8efafd767.png">

* User flipper to set `IS_USER_ELIGIBLE` pref propery to false
* Hide/show main screen
* Notice the same menu on `UserEligibilityErrorFragment`
<img width="468" alt="image" src="https://user-images.githubusercontent.com/4923871/187416126-2fbedb46-44cd-4063-854d-3e2d22d9d4d7.png">


##### Also, click through the app to verify that the dependency update does not break anything
